ops/terraform/services/migrator/deploy.groovy

```diff
@@ -138,4 +156,32 @@ boolean canMigratorDeploymentProceed(String sqsQueueName, String awsRegion) {
     }
 }
 
+// checks to determine whether the migrator deployment is required
+// returns true when the latest versioned migration script is newer
+// than the database's stored schema version
+// otherwise false
+boolean isMigratorDeploymentRequired(String bfdEnv, String awsRegion) {
+    println "Comparing schema migration versions..."
+    // Initialize stored schema version
+    int storedSchemaVersion = 0
+    // check SSM Parameter Store
+    try {
+        storedSchemaVersion = awsSsm.getParameter(
+                parameterName: "/bfd/${bfdEnv}/common/nonsensitive/database_schema_version",
+                awsRegion: awsRegion
+        ) as Integer
+    } catch(Exception ex) {
+        echo "Exception has been encountered getting the storedSchemaVersion from aws ssm, missing ssm parameter for stored schema version."
```

Comment:
let's use the value of `"/bfd/${bfdEnv}/common/nonsensitive/database_schema_version"` in this statement instead of `storedSchemaVersion`

Reply:
done